### ISSUE TITLE
Tea reader is broken with chrome 25

### ISSUE THREAD

**Issue by julien-duponchelle** [TEA-ebook/teabook-open-reader#29]
For your information i open a pull request about a bug with sticky and chrome 25:
https://github.com/alexmingoia/sticky/pull/15

**Comment by brianhanifin** [comment]
I just visited the demo site for the first time and the epub books never display on Chrome v25.0.1364.99. I tried several books, and even tried it in Incognito mode just to make sure my extensions weren't conflicting. [The same book](http://demo-open-reader.tea-ebook.com/ebook/epub#507584bd01034c69fe000001) opened up in Safari perfectly.

Here is an error from the Chrome Developer Console:

```
Uncaught TypeError: Type error reader-9b24449e2dc355b010c711fb6d566e7e.js:6
StickyStore.adapters.indexedDB.get reader-9b24449e2dc355b010c711fb6d566e7e.js:6
StickyStore.exec reader-9b24449e2dc355b010c711fb6d566e7e.js:6
StickyStore.get reader-9b24449e2dc355b010c711fb6d566e7e.js:6
u reader-9b24449e2dc355b010c711fb6d566e7e.js:6
(anonymous function)
```

Here is a screenshot of my Chrome console log.

![Screenshot_3_1_13_10_29_AM-2](https://f.cloud.github.com/assets/57336/210794/fbe49c42-829d-11e2-9201-7bc93a7fc112.png)
